src/Integration.UnitTests/Binding/BindCommandTests.cs

```diff
@@ -128,20 +128,26 @@ public void BindCommand_CanExecuteBind_VsState()
             ProjectViewModel projectVM = CreateProjectViewModel();
             BindCommand testSubject = this.PrepareCommandForExecution(new ConnectionInformation(new Uri("http://127.0.0.0")));
```

Comment:
The _PrepareCommandForExection_ method sets up the command and checks that it can execute: I'd have expected other tests to fail unless that method to set the additional required content.